images/clang_tidy/Dockerfile.erb

```
@@ -0,0 +1,15 @@
+FROM sider/devon_rex_base:2.21.0
```

[suggestion] Please bump the version as follows. See also #1278

```suggestion
FROM sider/devon_rex_base:2.22.0
```

---

Fixed. commit 617ce2886a8e07b498301eba8bbf0af051232d7d